src/main/java/com/google/api/codegen/config/LanguageGenerator.java

```diff
@@ -43,14 +44,17 @@
         Arrays.asList(new RewriteRule("^google(?!\\.cloud)", "google.cloud"));
     List<RewriteRule> pythonRewriteRules =
         Arrays.asList(new RewriteRule("^google(?!\\.cloud)", "google.cloud"));
+    List<RewriteRule> rubyRewriteRules =
+        Arrays.asList(new RewriteRule("^google(?!\\.cloud)", "google.cloud"));
```

Comment:
With three instances of this RewriteRule, I think it's worth creating it once and passing it into each of the three languages.